notebooks/time-to-merge-prediction/workshop/intro_to_jupyternotebooks.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.    print(s3_access_key)
</pre><p>can we remove print statements for s3_access_key and s3_secret_key. We might have to switch to an AWS bucket if issues arise with the current bucket. This creates a risk of exposing these creds.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
fixed